tests/network/Proxy.test.ts

```diff
@@ -311,16 +312,16 @@ describe(Proxy.name, () => {
     ).rejects.toThrow(networkErrors.ErrorConnectionStartTimeout);
     expect(receivedCount).toBe(1);
     // Can override the timer
-    const timer = timerStart(2000);
+    const timer = new Timer({ delay: 1000 });
     await expect(() =>
       proxy.openConnectionForward(
         nodeIdABC,
         localHost,
         utpSocketHangPort as Port,
-        timer,
+        { timer },
       ),
     ).rejects.toThrow(networkErrors.ErrorConnectionStartTimeout);
-    timerStop(timer);
+    timer.cancel('clean up');
```

Comment:
We are going to change this in a later PR, so this can be rollbacked to how it was. Integrating timers all over the codebase.

Reply:
I've reverted this. This change is left to the wider `timedCancellable` changes PR.